curve25519/src/lib.rs

```diff
@@ -6,8 +6,8 @@ pub use libcrux_hacl_rs::curve25519_51 as hacl;
 #[cfg(feature = "hacl")]
 mod impl_hacl;
 
-#[cfg(feature = "portable_hacl")]
-pub use impl_hacl::HaclCurve25519 as Impl;
+#[cfg(feature = "hacl")]
```

Comment:
Do we need this in any other libcrux crate? If yes, we should start using a different feature for it. If not, let's not expose it at all.